### PR TITLE
Update vulkan-validation-layers/spirv-tools to 1.3.261/2023.4rc2

### PR DESCRIPTION
**Pull request by MehdiChinoune**
@kasper93

### CODE REVIEW

**Review by lazka — APPROVED** [review]
lgtm

File: mingw-w64-spirv-tools/PKGBUILD [hunk 1]
```diff
@@ -19,15 +22,8 @@ makedepends=("${MINGW_PACKAGE_PREFIX}-cc"
              "${MINGW_PACKAGE_PREFIX}-ninja"
              "${MINGW_PACKAGE_PREFIX}-python"
              "${MINGW_PACKAGE_PREFIX}-spirv-headers")
-source=("${_realname}-${pkgver}.tar.gz::https://github.com/KhronosGroup/SPIRV-Tools/archive/sdk-${pkgver}.tar.gz"
-        002-Do-not-statically-link-to-the-C-runtime-with-mingw.patch)
```
Comment (lazka) [inline mingw-w64-spirv-tools/PKGBUILD:23]: oh, was that dropped on purpose? the code still seems to be there upstream

Comment (MehdiChinoune) [reply mingw-w64-spirv-tools/PKGBUILD:23]: I will take a look.

**Comment by kasper93** [comment]
I'm not a big fan of switching to RC (or beta) versions for main repo. But if it works and you are ok with it, LGTM too.

**Comment by kasper93** [comment]
> Better than reverting to previous version.

Why? What's wrong with stable and tested version?

**Comment by lazka** [comment]
@MehdiChinoune hm?

**Comment by MehdiChinoune** [comment]
What?

**Comment by lazka** [comment]
Would it be a good idea to follow the sdk release versions in the future (https://vulkan.lunarg.com/sdk/latest.json), to avoid similar problems? Or are there any reasons why we should update single components independently?

**Comment by MehdiChinoune** [comment]
Should we revert all?

**Comment by kasper93** [comment]
> Would it be a good idea to follow the sdk release versions in the future (https://vulkan.lunarg.com/sdk/latest.json), to avoid similar problems? Or are there any reasons why we should update single components independently?

Vulkan SDK releases are pretty slow. We can update those packages sooner. I would just stick to some stable tags and if something is not building correctly just wait for new release instead of chasing latest version possible.

> Should we revert them all?

IMHO, at this point keep it as is.

**Pull request merged**